sparsnas_decode.cpp

```diff
@@ -195,24 +196,30 @@ class SignalDetector {
           error_sum += fabs(freq);
           error_sum_count += 1;
         }
+      } else {
+      m += sprintf(m, "{\"CRC\": \"ERR\"");
       }
 
-      m += sprintf(m, (crc == packet_crc) ? ",\"CRC\":\"ok\"" : ",\"CRC\": \"ERR\"");
```

Comment:
What happens to the json when this is removed and replaced with 

` m += sprintf(m, "{\"CRC\": \"ERR\"");`

Reply:
The code 
```
m += sprintf(m, "{\"CRC\": \"ERR\"");
```
is in a second if statement. so if crc error the json would look like this {"CRC":"ERR","Sensor":653213} published to the crc topic.
And if not then the json would look like normal published to the normal topic.

Reply:
<img width="1136" alt="skarmavbild 2018-02-26 kl 11 07 07" src="https://user-images.githubusercontent.com/2168003/36664520-405ceebc-1ae5-11e8-8ff2-ace6e02bad9a.png">